### DISCUSSION TITLE
[@types/aws-lambda] Dynamo Streams handler missing fields

### DISCUSSION THREAD

**Discussion by jacsamell** [DefinitelyTyped/DefinitelyTyped#58597] [Issues with a @types package]
The following fields need adding to the dynamo streams handler:

type DynamoDBStreamEvent = {
  ...,
  window?: { start: string, end: string },
  state?: object,
  isFinalInvokeForWindow?: boolean,
  isWindowTerminatedEarly?: boolean
}

type DynamoDBStreamEventHandler = Handler<DynamoDBStreamEvent, { state: object } | void>

See: https://docs.aws.amazon.com/lambda/latest/dg/with-ddb.html

[a comment by an automated account is not shown]

**Comment by darbio** [comment]
Hi mate. Feel free to submit a PR with these changes if you need them added in.

**Comment by jacsamell** [comment]
https://github.com/DefinitelyTyped/DefinitelyTyped/pull/58600
